src/robobo/simulation.py

```diff
@@ -351,11 +361,22 @@ def collected_food(self):
         return ints[0]
 
     def base_position(self):
+        """Get the position of the green target zone (for task3)."""
         return vrep.unwrap_vrep(
             vrep.simxGetObjectPosition(self._clientID, self._base, -1, vrep.simx_opmode_blocking)
         )
     
     def base_detects_food(self):
+        """Check whether the red block is in the green target zone (for task 3)."""
         detection, _detection_point, _detected_handle, _detected_normal \
             = self._vrep_read_proximity_sensor(self._base, vrep.simx_opmode_buffer)
-        return bool(detection)
\ No newline at end of file
+        return bool(detection)
+
+    def puck_position(self):
+        """Return position of the red block for task3."""
+        puck_handle = self._vrep_get_object_handle(
+            "Food".format(self._value_number), vrep.simx_opmode_blocking
```

Comment:
what is this format suppose to do? As far as I can tell it's changing nothing here (I'm surprised it doesn't give out errors)